src/data/builder-tools.js

```diff
@@ -53,6 +53,15 @@ export const Tags = {
     color: '#921f32',
   },
 
+  // Crystal
+  crystal: {
+    label: "Crystal",
+    description:
+      "Crystal language",
+    icon: null,
+    color: '#000',
```

Comment:
this colour is already in use by `chainindex` ... can you pick another one (randomly or however it suits you) that seems visually distinct from the other colours?

Reply:
Their style guides suggest using either `#000`, `#fff` or `#777`:

https://crystal-lang.org/media/

Since white is invisible on a light background and `#777` is visually too close to **HTTP**, I've opted to use `#ddd`. It reads white-ish and it's visually distinct from anything else. It also works well on both the light as well as the dark theme:

![image](https://user-images.githubusercontent.com/107324/225241405-ff675852-3543-4cca-a53d-42e399ab5efa.png)

![image](https://user-images.githubusercontent.com/107324/225242029-e03037fe-541f-4d21-a4a2-a15c7ff316ac.png)